### DISCUSSION TITLE
Adding a new gpkg (glibc packages) repo

### DISCUSSION THREAD

**Discussion by Maxython** [termux/termux-packages#16692] [Polls]
**This poll is for Termux developers only!!!**

I'm going to be developing a `gpkg` repo (glibc packages), but I've run into requests for this repo to be added to the official Termux repository. The `gpkg-dev` repo is currently fully developed and maintained by the @termux-pacman org. **For understanding:** `gpkg-dev` - this repo stores glibc packages for testing and compiling other glibc packages, `gpkg` - this repo stores the final assembly of glibc packages that are compiled thanks to `gpkg-dev`.

In theory, need to "configure" the `build-package.sh` compiler and create a new script for GitHub Actions (but perhaps something else is needed).

**If the majority is in favor of the additions,** then I wanted to understand who would be willing to maintain glibc packages (other than me).

**If the majority is against the addition,** then the `gpkg` repo will be developed in the @termux-pacman org (PS: a command will be created in the @termux-pacman org to support glibc-packages). Perhaps the possibility of `gpkg` support for `apt` will be considered, but it depends on which build structure will be used.

If you have any questions, ask!

**Comment by sylirre** [comment]
I'm ok with this as soon as:

1. GNU libc packages build system is maintained as separate Git repository within @termux.
2. All GNU libc packages have suffix `-glibc` with NO exception.
3. All GNU libc packages hosted by @termux are in .deb format only (we don't provide hosting for formats other than Debian).
4. `termux-packages` has a `glibc-repo` package to enable subscription to GNU libc packages apt repository.

P.S. It looks like voting is not possible when discussion is locked.

**Reply by Maxython** [reply]
> GNU libc packages build system is maintained as separate Git repository within @termux.

Do you want to create a new GitHub repo (something like `termux/glibc-packages`)?

> All GNU libc packages have suffix -glibc with NO exception.

This sorting of packages is already implemented in `gpkg-dev` and will be implemented in `gpkg` as well.

> P.S. It looks like voting is not possible when discussion is locked.

Yes, you are right. Everything is fixed.

**Reply by sylirre** [reply]
Yes, I can create termux/glibc-packages and give you admin access for this repository.

GNU libc stuff shouldn't be in `termux-packages`, unless there is a good reason (which I don't know) for adding it here.

**Reply by Maxython** [reply]
Ok, I got you.

There are no such good reasons to support gpkg in termux-packages. I think your idea is even better.

**Reply by sylirre** [reply]
Done, https://github.com/termux/glibc-packages

**Reply by Maxython** [reply]
Wow, so fast /j

All right, let's decide which repo will be the "mirror" (it's `termux/glibc-packages` or it's `termux-pacman/glibc-packages`). For understanding, by the word "mirror" I mean a repo that automatically adds commits from the root repo (for example: repo `termux-pacman/termux-packages` is mirror).

I suggest that `termux/glibc-packages` be a mirror and `termux-pacman/glibc-packages` the root repo, I think this will speed up the development process.

**Reply by sylirre** [reply]
Ok, let's make `termux/glibc-packages` a mirror
